resources/views/registerProject.blade.php

```diff
@@ -0,0 +1,218 @@
+			</div>
+		</div>
+	</div>
+</div>
+
+@section('jquery')
+<!-- Internal jQuery -->
+<script>
+	/* Search for Skillsets */
+	var skillsets = [
+		{ title: 'Skillset 1' },
+		{ title: 'Skillset 2' },
+		{ title: 'Skillset 3' },
+		{ title: 'Skillset 4' },
+		{ title: 'Skillset 5' },
+	];
+	$('.ui.search').search({source: skillsets});
+	$('.ui.search.category.dropdown').dropdown({useLabels: false});
+	$('.imgUploader').click(function(event) {
+		event.preventDefault();
+		$('#imgInput').click();
+	});
+
+	/*
+	 Uploading Image function:
+	function readURL(input) {
+		  if (input.files && input.files[0]) {
+			  var reader = new FileReader();
+
+			  reader.onload = function (e) {
+				  $('#blah').attr('src', e.target.result);
+			  }
+
+			  reader.readAsDataURL(input.files[0]);
+		  }
+	  }
+	  $("#imgInp").change(function(){
+			  readURL(this);
+	   });
+
+	  HTML for funtcion above:
+	  <form id="form1" runat="server">
+			<input type='file' id="imgInp" />
+			<img id="blah" src="#" alt="your image" />
+		</form>
+
+		HTML AHORITA:
+		<div class="projectImg">
+			<img src="https://lorempixel.com/400/400" alt="Project Image" class="ui centered medium circular image"/>
+			<a href="#" class="imgUploader">
+				<button id="uploadImage" class="fluid ui button">Upload Image</button>
+			</a>
+			<input type="file" id="imgInput" style="display:none" multiple accept='image/*'>
+		</div>
+	  */
+
+
+	/* Register Button Click */
+	$('#registerProject').on('click', function(event){
+		//event.preventDefault();
+		//Alert Message
+		let $errorMessage = $('#errorMessage');
+
+		// Project Name
+		let $projectName = $('#projectName');
+		if($projectName.val() == ""){
+			$projectName.addClass('field error');
+			$errorMessage.removeClass('hidden');
+		}
+
+		// Team Name
+		let $teamName = $('#teamName');
+		if($teamName.val() == ""){
+			$teamName.addClass('field error');
+			$errorMessage.removeClass('hidden');
+		}
+
+		// Category Dropdown
+		let $category = $('category');
+		if($category.val() == 0){
+			$category.addClass('field error');
+			$errorMessage.removeClass('hidden');
+		}
+
+		// Skillsets
+
+		// Public Milestones
+		let $milestones = $('#milestones');
+		if($milestones.val() == ""){
+			$milestones.addClass('field error');
+			$errorMessage.removeClass('hidden');
+		}
+
+		// Short Description
+		let $shortDescription = $('#shortDescription');
+		if($shortDescription.val() == ""){
+			$shortDescription.addClass('field error');
+			$errorMessage.removeClass('hidden');
+		}
+
+		// Long Description
+		let $longDescription = $('#longDescription');
+		if($longDescription.val() == ""){
+			$longDescription.addClass('field error');
+			$errorMessage.removeClass('hidden');
+		}
+
+		// Video Pitch
+		let $videoPitch = $('#videoPitch');
+		if($videoPitch.val() == ""){
+			$videoPitch.addClass('field error');
+			$errorMessage.removeClass('hidden');
+		}
+	});
+</script>
+@endsection
```

Comment:
why is @endsection duplicated?

Reply:
No está duplicado, uno es para `content` y otro es para `jquery` sections.